CSS/media.css

```diff
@@ -90,4 +88,46 @@
   .menu {
     display: none;
   }
+
+  .pop-content {
+    margin-left: 250px;
+    padding: 12px;
+    display: flex;
+    flex-direction: column;
+    justify-items: center;
+    width: 650px;
+    max-height: 600px;
+    background: #fff;
+    border: 1px solid #ebecf0;
+    box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+    border-radius: 16px;
+  }
+
+  .popimg {
+    width: 100%;
+    height: 300px;
+    margin-bottom: 20px;
+  }
```

Comment:
- [ ] [REQUIRED] Good job so far! There are design issues with your Popup. Please, in desktop and mobile versions, your popup does not open on full screen. See the problem below:

- ### [HINT] Please watch [this tutorial](https://youtu.be/MBaw_6cPmAw)!

- Desktop version.

![image](https://user-images.githubusercontent.com/63915024/181104924-92770670-3da9-4b06-93e1-d40598bcb11e.png)

- Mobile version.

![image](https://user-images.githubusercontent.com/63915024/181104980-6edb186a-2a82-4c5e-ace6-f48c66cf9440.png)